### ISSUE TITLE
Trigger on Button in AppBar is executed multiple times

### ISSUE THREAD

**Issue by tibel** [jlaanstra/Windows.UI.Interactivity#9]
After each open/close of the AppBar the command is executed one more time. It seems that another event registration is added on each open of the AppBar.

``` xml
<Page.BottomAppBar>
  <AppBar VerticalAlignment="Bottom" x:Name="TestAppBar" IsSticky="True">
    <Grid>
    <Button x:Name="AppBarButton3" Style="{StaticResource EditAppBarButtonStyle}">
    <i:Interaction.Triggers>
    <i:EventTrigger EventName="Click">
    <i:InvokeCommandAction Command="{Binding AppBarButtonCommand}" />
    </i:EventTrigger>
    </i:Interaction.Triggers>
    </Button>
    </Grid>
  </AppBar>
</Page.BottomAppBar>
```

**Comment by jmclain** [comment]
hello jlaanstr,

Almost fixed in 37eb96e. However when calling RemoveEventHandler<T>() you have to use the same delegate that was passed to AddEventHandler<T>() for the RemoveMethod. At least that works for me.

https://github.com/jlaanstra/Windows.UI.Interactivity/issues/8

Thanks for all the work.

joe mclain

**Comment by jlaanstra** [comment]
Fixed in 37eb96e5de0601c307a670a126666a8ef6ad5363.

**Comment by jlaanstra** [comment]
Not yet fixed.
